### ISSUE TITLE
coordinates error

### ISSUE THREAD

**Issue by g101418** [marchah/sea-ports#3]
there are many problem in ports coordinates
such as : CNYTN
it should be in Guangdong China, but the coordinates are in Fujian China

**Comment by marchah** [comment]
@g101418 yes i'm aware the coordinates are not all accurate, forgot where the data is coming from.

If you know a good place to get a data dump with proper coordinates i will fix them.

For now i will just add a warning in the README

**Comment by GeorgFerdinandSchneider** [comment]
Hi @marchah !

I was looking into the data on seaports. I was thinking it might make sense to combine or augment the dataset with [wikidata](wikidata.org) e.g. the coordinates for Ajman  are given:

https://www.wikidata.org/wiki/Q530171

Wikidata can be queried through a SPARQL service. Maybe thats the data dump you are looking for?

Are the coordinates the ones of the harbor or of the city ?

Best

Georg

**Comment by marchah** [comment]
@GeorgFerdinandSchneider i think the data dump is originally coming from this website: https://www.unece.org/cefact/locode/service/location.html

The coordinate looks to be the city ones.

Unfortunately i currently don't have the time to check on how to improve the data and every improvement is welcome :)
